views/login.ejs

```diff
@@ -1,24 +1,36 @@
 <!DOCTYPE html>
 <html>
-
 <head>
-
-	<title>Login | CodeManiacs</title>
-	<meta name="viewport" content="width=device-width, initial-scale=1">
-	<link rel="stylesheet" type="text/css" href="/css/sheet.css">
-	<link href='https://fonts.googleapis.com/css?family=Sofia' rel='stylesheet'>
+<title> Login | CodeManiacs</title>
+<meta name="viewport" content="width=device-width, initial-scale=1">
+<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+<script type="application/x-javascript"> addEventListener("load", function() { setTimeout(hideURLbar, 0); }, false); function hideURLbar(){ window.scrollTo(0,1); } </script>
+<link href="./style.css" rel="stylesheet" type="text/css" media="all" />
+<link href="//fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i" rel="stylesheet">
 </head>
-
 <body>
-	<header>
-		<% include partials/header %>
-    </header>
-    
-    <h1>Login</h1>
-
-	<footer>
-		<% include partials/footer %>
-	</footer>
+	<div class="main-w3layouts wrapper">
+		<h1><b><u> SignIn CodeManiacs</u></b></h1>
+		<div class="main-agileinfo">
+			<div class="agileits-top">
+				<form action="#" method="post">
+					<input class="text email" type="email" name="email" placeholder="Email" required="">
+					<input class="text" type="password" name="password" placeholder="Password" required="">
+					<div class="wthree-text">
+						<div class="clear"> </div>
+					</div>
+					<input type="submit" value="SIGNUP">
+				</form>
+				<p>Don't have an Account? <a href="./signup.ejs"> SignUp Now!</a></p>
```

Comment:
It should be `<a href="/user/signup">`

Reply:
Sir, this means I have to create a new folder named user and place the files further in it??

Reply:
No. The route for accessing this page is already http://localhost:3000/user/login and http://localhost:3000/user/signup Just need to change the href

Reply:
Run the app and click on Login. You can see the URL